src/components/View/Struttura/StrutturaSchemaOrg.jsx

```diff
@@ -0,0 +1,69 @@
+import { toPublicURL } from '@plone/volto/helpers';
+import {
+  SchemaOrgUtils,
+  richTextHasContent,
+  SchemaOrg,
+} from 'io-sanita-theme/helpers';
+
+const StrutturaSchemaOrg = ({ content }) => {
+  let siteTitle = SchemaOrgUtils.getSiteTitle();
+
+  let schemaOrg = {
+    '@type': 'MedicalOrganization',
+    url: toPublicURL(content['@id']),
+    name: siteTitle,
+    additionalType: content.tipologia_struttura?.title,
+  };
+  let description = [];
+
+  if (content.description) {
+    description.push(content.description);
+  }
+
+  if (richTextHasContent(content.descrizione_estesa)) {
+    description.push(
+      SchemaOrgUtils.fieldDataToPlainText(content.descrizione_estesa),
+    );
+  }
+  console.log(content);
+  if (content.street || content.city || content.zip_code) {
```

Comment:
più che un OR metterei tutti AND perchè se passa solo con uno poi gli altri si rompono

Reply:
Sono campi obbligatori, quindi non si spaccheranno. Quello che possiamo fare, volendo, è anche togliere la condizione. L’ho messa più che altro per sicurezza

Reply:
L'ho fatto, tolto la condizione e messo dentro al schemaOrg diretamente